controllers/operands/kubevirt.go

```diff
@@ -128,8 +125,8 @@ var (
 		kvHotplugVolumesGate,
 		kvExpandDisksGate,
 		kvGPUGate,
+		kvDownwardMetrics,
```

Comment:
Should not be in this list. This list is the fg that hco always sets in kv.

Reply:
See: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2823#discussion_r1505640247

Reply:
Done.